examples/serviceExpiration/api_get_service_that_expired_soon.md

```diff
@@ -0,0 +1,64 @@
+How to get services that will expired soon with python wrapper?
+---------------------------------------------------------------
+
+This documentation will help you to know what services will be expired soon and need to be renew. Following script will check each of your OVH services and check expiration date.
```

Comment:
- help you to list* ? That only semantic.
- The following script will check the expiration date of each services attached to your consumer_key